adminapi/src/main/java/io/minio/admin/MinioAdminClient.java

```diff
@@ -97,6 +103,9 @@ public String toString() {
   private static final MediaType DEFAULT_MEDIA_TYPE = MediaType.parse("application/octet-stream");
   private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
 
+  private static final Pattern SERVICE_ACCOUNT_NAME_REGEX =
+      Pattern.compile("^[a-zA-Z][a-zA-Z0-9_-]*");
```

Comment:
`public static final Pattern SERVICE_ACCOUNT_NAME_REGEX = Pattern.compile("^(?!-)(?!_)[a-z_\\d-]{1,31}(?<!-)(?<!_)$", Pattern.CASE_INSENSITIVE);` is much better regex